Components/Avatar/Avatar.css

```diff
@@ -0,0 +1,95 @@
+.small-avatar{
+    height: 7rem;
+    width: 7rem;
+    border-radius: 10rem;
```

Comment:
You can define a global variable in the style guide file for border-radius

Reply:
Done thanks!